dapp_stats/statistics/models.py

```diff
@@ -0,0 +1,32 @@
+from dataclasses import dataclass, field
+from datetime import datetime, timedelta
+from typing import Dict
+
+from dapp_stats.statistics.enums import NodeState
+
+
+@dataclass
+class NodeStat:
+    timestamp: datetime
+    state: NodeState
+    _changes: int = 1
+    _launched_successfully: bool = False
+    _working_time: timedelta = field(default_factory=timedelta)
+    _time_to_launch: timedelta = field(default_factory=timedelta)
```

Comment:
I honestly don't understand the usage of `_time_to_launch` ... the name and especially the `to_dict` key suggests it's some kind of an ETA but:
* it's initial value is an empty timedelta -> which means the "ETA" starts with "immediately" or 0 seconds... I don't think that's what we want ... I'd rather have `None` returned here in such a case
* when it's actually set -> it's not an estimate but the exact time that it took to launch... 

if it's supposed to be an ETA, I believe the logic for setting should somehow reflect that... 

and if it's _not_ an ETA, please change the output key to `time_to_launch` or just `startup_time` so that it does _not_ suggest it's some kind of forward-looking estimate... 

plus, in both cases, setting it to 0 seconds initially doesn't make too much sense...